wist-scoreboard-generator/package.json

```diff
@@ -11,10 +11,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "~12.1.0-",
+    "@angular/cdk": "^12.2.13",
```

Comment:
please do an `ng update @angular/cli@^12 @angular/core@^12`. See https://angular.io/cli/update for more details. The version of the packages used are pretty old.

Reply:
Just an FYI the current version of Angular is 13 but angular 12 has support until November. See: https://angular.io/guide/releases#support-policy-and-schedule